packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -263,19 +281,51 @@ export default class Meetings extends WebexPlugin {
         StaticConfig.set(this.config);
         LoggerConfig.set(this.config.logging);
         LoggerProxy.set(this.webex.logger);
-        if (this.webex.canAuthorize) {
+        Trigger.trigger(
+          this,
+          {
+            file: 'meetings',
+            function: 'onReady'
+          },
+          EVENT_TRIGGERS.MEETINGS_READY
+        );
+      });
+    }
+
+    /**
+     * Explicitly sets up the meetings plugin by registering
+     * the device, connecting to mercury, and listening for locus events.
+     *
+     * @returns {Promise}
+     * @public
+     * @memberof Meetings
+     */
+    register() {
+      if (!this.webex.canAuthorize) {
+        LoggerProxy.logger.error('meetings->register#ERROR, Unable to register, SDK cannot authorize');
+
+        return Promise.reject(new Error('SDK cannot authorize'));
+      }
+
+      return this.webex.internal.device.register()
+        .then(() => this.webex.internal.mercury.connect())
+        .then(() => {
           this.listenForEvents();
```

Comment:
Does the register happens when plugin is loaded ? if yes should `listenForEvents` be here

Reply:
Register does not happen automatically with the plugin. The issue that this PR is attempting to resolve is due to the fact that the meetings plugin automatically will listen to events from locus, even if the SDK user isn't concerned with them. Because meetings plugin is part of the main "webex" package, we need to make things like listening for events less automatic and more direct.